packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -115,16 +139,22 @@ class FlutterCommandRunner extends CommandRunner<void> {
     if (verboseHelp) {
       argParser.addSeparator('Options for testing the "flutter" tool itself:');
     }
-    argParser.addFlag('show-test-device',
+    argParser.addFlag(FlutterGlobalOptions.kShowTestDeviceFlag,
         negatable: false,
         hide: !verboseHelp,
         help: 'List the special "flutter-tester" device in device listings. '
               'This headless device is used to test Flutter tooling.');
-    argParser.addFlag('show-web-server-device',
+    argParser.addFlag(FlutterGlobalOptions.kShowWebServerDeviceFlag,
         negatable: false,
         hide: !verboseHelp,
         help: 'List the special "web-server" device in device listings.',
     );
+    argParser.addFlag(
+      FlutterGlobalOptions.kContinuousIntegrationFlag,
```

Comment:
if this is a global argument, does it matter in what order it appears when it's invoked on a sub-command?

Reply:
If you mean like `flutter build --ci ios` vs `flutter build ios --ci`, no it doesn't matter.

Reply:
cool